app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
-end
+    def full_title page_title
+        base_title = "Ruby on Rails Tutorial Sample App"
+        page_title.blank? ? base_title : "#{page_title} | #{base_title}"
+      end
+    end
```

Comment:
![Uploading image.png…]()

Reply:
sorry, the comment says "Uploading image.png…" . I cannot see what need to be fix

Reply:
done